content/acra/configuring-maintaining/general-configuration/acra-translator.md

```diff
@@ -315,18 +315,6 @@ acra-keymaker \
     --keys_output_dir=/tmp/translator_keys
 ```
 
-### Transport keys
-
-Used in `AcraConnector ↔︎ AcraTranslator` connection with `SecureSession` as transport encryption.
-
-```
-acra-keymaker \
-    --keystore=v2 \
-    --client_id=client1 \
-    --generate_acratranslator_keys \
-    --keys_output_dir=/tmp/translator_keys
-```
-
```

Comment:
I would not remove these, just add a warning label "AcraConnector and its transport keys are deprecated since 0.91 and removed since 0.92. Consider migrating to TLS instead."